docker.go

```diff
@@ -817,7 +818,7 @@ func (c *DockerContainer) copyLogsTimeout(stdout, stderr io.Writer, options *con
 		// Timeout or client connection closed, retry.
 	default:
 		// Unexpected error, retry.
-		Logger.Printf("Unexpected error reading logs: %v", err)
+		log.Printf("Unexpected error reading logs: %v", err)
```

Comment:
bug: not new but this should use c.logger
```suggestion
		c.logger.Printf("Unexpected error reading logs: %v", err)
```